### PR TITLE
Ja art19 proxy

### PR DESCRIPTION
**Pull request by jaslakson**
using proxy server url and multiple requests to art19 to finally get the image asset.

### CODE REVIEW

File: src/components/Cards/PodcastEpisodeCard/PodcastEpisodeCard.stories.js [hunk 1]
```diff
@@ -4,6 +4,17 @@ import fetch from 'cross-fetch';
 
 import PodcastEpisodeCard from './index';
 
+const headers = {
+  method: 'GET',
+  headers: {
+    'Accept': 'application/vnd.api+json',
+    'Authorization': 'Token token="jQ-ldsLx7USTQ4mjtOEBPIOsEXAno8UTh8l2KomNLQM5B75NUL-P9iFVGd1lF6c9-Lcq1dmUFTmhZTBsb09BmA", credential="14c39803-3e99-47c4-a4d3-d79398e74089"',
```
Comment (ccandelora) [inline src/components/Cards/PodcastEpisodeCard/PodcastEpisodeCard.stories.js:11]: is there anyway to store these values in env variables?

File: src/components/Cards/PodcastEpisodeCard/PodcastEpisodeCard.stories.js [hunk 2]
```diff
@@ -4,6 +4,17 @@ import fetch from 'cross-fetch';
...
+  },
+  mode: 'cors',
+};
+
+const proxyUrl = 'https://www-test.americastestkitchen.com/art19';
```
Comment (ccandelora) [inline src/components/Cards/PodcastEpisodeCard/PodcastEpisodeCard.stories.js:16]: Not sure if you want to point to a local env here.

Comment (ccandelora) [reply src/components/Cards/PodcastEpisodeCard/PodcastEpisodeCard.stories.js:16]: maybe make this an env variable too.

Comment (jaslakson) [reply src/components/Cards/PodcastEpisodeCard/PodcastEpisodeCard.stories.js]: @ccandelora Good call - I updated as staging for now. I will look into the environment variable thing as well.

**Pull request merged**